src/components/timeline/LayerActivity.svelte

```diff
@@ -110,9 +109,7 @@
       const x = offsetX - dragOffsetX;
       const unixEpochTime = xScaleView.invert(x).getTime();
       const start_time_doy = getDoyTime(new Date(unixEpochTime));
-      if (unixEpochTime !== dragPoint.x) {
-        effects.updateActivityDirective(dragPoint.id, { start_time_doy });
-      }
+      effects.updateActivityDirective(dragPoint.id, { start_time_doy });
```

Comment:
Tiny nit - without this time equality check you do get activity updates when just clicking on an activity in the timeline even if you didn't drag. Not a big deal though.

Reply:
Actually, when you just click on an activity it updates which is probably not desired.

Reply:
I'm not seeing an update when you click. I'm seeing `effectiveActivityArguments` and `validateActivityArguments` called which is expected since we need to grab that data separately when we select an activity. Are those the requests you are seeing or are you seeing an actual update?

Reply:
Ah I think I se what you are saying. It only updates when you unlock. I'll add a new check to prevent the update!

Reply:
Pulled and no longer see this behavior, must have been some fix in the last 30 mins.

Reply:
I wrapped the check anyway to make sure it never happens because I saw it occur in unlocked drag mode. Thanks for the suggestions.

Reply:
Yep just realized that's the situation I was seeing it in.